build/pandoc-svg.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
+"""
+Pandoc filter to convert svg files to pdf as suggested at:
+https://github.com/jgm/pandoc/issues/265#issuecomment-27317316
+"""
+
+__author__ = "Jerome Robert"
+
+import mimetypes
+import subprocess
+import os
+import sys
+from pandocfilters import toJSONFilter, Para, Image
+
+fmt_to_option = {
+    "docx": ("--export-eps", "eps")
+}
+
+
+buildir = os.getcwd()
+
+
+def svg_to_any(key, value, fmt, meta):
+    if key == 'Image':
+        if len(value) == 2:
+            # before pandoc 1.16
+            alt, [src, title] = value
+            attrs = None
+        else:
+            attrs, alt, [src, title] = value
+        mimet, _ = mimetypes.guess_type(src)
+        option = fmt_to_option.get(fmt)
+        if mimet == 'image/svg+xml' and option:
+            base_name, _ = os.path.splitext(src)
+            eps_name = os.path.join(buildir, base_name + "." + option[1])
+            file_name = os.path.join(buildir, src)
+            try:
+                mtime = os.path.getmtime(eps_name)
+            except OSError:
+                mtime = -1
+            if mtime < os.path.getmtime(src):
+                cmd_line = ['inkscape', option[0], eps_name, file_name]
```

Comment:
Does this method depend on inkscape? That's a pretty heavy dependency? Can inkscape be installed via conda?